ark/settings.py

```diff
@@ -0,0 +1,18 @@
+# hope u liek capital letters
+
+# default segmented csv column names
+CELL_SIZE = 'cell_size'                     # cell size
+CELL_LABEL = 'label'                        # cell label number (regionprops)
+AREA = 'area'                               # cell area (regionprops)
+ECCENTRICITY = 'eccentricity'               # cell eccentricity (regionprops)
+MAJ_AXIS_LENGTH = 'major_axis_length'       # cell major axis length (regionprops)
+MIN_AXIS_LENGTH = 'minor_axis_length'       # cell minor axis length (regionprops)
+PERIMITER = 'perimeter'                     # cell perimiter (regionprops)
```

Comment:
We'll probably need to wait until next Friday's meeting to merge this in. I'm sure Mike will have opinions about what the default names should be